corgi/core/models.py

```diff
@@ -515,13 +519,16 @@ def manifest(self) -> str:
         """Return an SPDX-style manifest in JSON format."""
         return ProductManifestFile(self).render_content()
 
-    def get_latest_components(self):
-        """Return root components from latest builds."""
+    def get_latest_components(self, start_dt=now()):
+        """Return root components from latest builds (or from a supplied start datetime)."""
         root_components = (
             Q(type=Component.Type.SRPM)
             | Q(type=Component.Type.CONTAINER_IMAGE, arch="noarch")
             | Q(type=Component.Type.RHEL_MODULE)
         )
+        dt_range = Q()
+        if start_dt:
+            dt_range &= Q(software_build__completion_time__lte=start_dt)
```

Comment:
This seems counterintuitive: start time filters components that have been built *before* the specified datetime? I would expect it to return everything built *after* (`gte`) that datetime? Or am I reading this wrong?

Reply:
yes, it is confusing ... Ricardo used start ... it could just as easily be date_before and we could contemplate adding a date_after ... will ping you/Jason for further thoughts

Reply:
not urgent, will remove